python/graphstorm/model/lm_embed.py

```diff
@@ -305,10 +316,17 @@ def _save_embeddings(self):
             embed_path = os.path.join(os.path.join(
                     os.path.join(self._embed_path, "lm_cache"), ntype),
                     self._get_model_name(ntype))
-            save_pytorch_embedding(embed_path,
-                                   self._lm_emb_cache[ntype],
-                                   get_rank(),
-                                   get_world_size())
+            if self.use_wg:
+                save_wholegraph_embedding(embed_path,
+                                        self._lm_emb_cache[ntype],
+                                        get_rank(),
+                                        get_world_size(),
+                                        fmt="binary")
```

Comment:
Currently, I coded the embedding saved in `fmt="binary"` format. Let me know if you want it to be "pytorch" as default. @classicsong

Reply:
I think "binary" works.